docs/notebooks/usage.ipynb

```diff
@@ -8,9 +8,7 @@
     "\n",
```

Comment:
<p>I think "Units Handling" is more appropriate than "Units handling", since "Units Handling" is a page with its own title, and also is a noun phrase, handling isn't just a normal participle</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>